src/main/java/TaskManager.java

```diff
@@ -2,40 +2,68 @@ public class TaskManager {
     private Task[] tasks = new Task[100];
     private int tasksCount = 0;
 
+    public void printAcknowledgement(){
+        String acknowledgement = "____________________________________________________________\n"
+                + "Got it. I've added this task: \n"
+                + " " + tasks[tasksCount].toString();
+        System.out.println(acknowledgement);
+        this.tasksCount++;
+        System.out.println("Now you have " + this.tasksCount + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public void handleTaskWithTime(String type, String description){
```

Comment:
Naming of method seems ok. Maybe can add a short overview of the method so as to provide reviewers a better idea on what it does!